compiler-rt/test/profile/Linux/instrprof-weak-symbol.c

```diff
@@ -0,0 +1,16 @@
+// Test the linker feature that treats undefined weak symbols as null values.
+
+// RUN: %clang_pgogen -o %t %s
```

Comment:
This just occurred to me, if the tested feature relies on linker implementation, does it need `REQUIRES: <platform>` so that `not %t` passes?

Reply:
Please discard this comment. 

@snehasish pointed out build-bots doesn't really have coverage of AIX linker, and the executable returns 1 if weak symbol is not null.

Reply:
Ah no worries! I actually tried running this on AIX and the test is unsupported. I think compiler-rt [checks](https://github.com/llvm/llvm-project/blob/d933b88b71b00461815667d7cd0bb2fecd8606db/compiler-rt/test/profile/Linux/lit.local.cfg.py#L45) to see if the host is Linux before running the tests in the `Linux` folder.